python/tvm/autotvm/tuner/tuner.py

```diff
@@ -174,6 +176,13 @@ def tune(self, n_trial, measure_option, early_stopping=None, callbacks=(), si_pr
             else:
                 logger.setLevel(old_level)
 
+        if error_ct == i:
+            logging.warning(
+                "Could not find a single schedule for task %s. A fallback config will be used. "
```

Comment:
```suggestion
                "Could not find any valid schedule for task %s."
```
IMHO, "A fallback config will be used" is not a proper message here and could be removed, because AutoTVM doesn't really compile a model. When you are really compiling the model, you will see the corresponding message.